build.gradle

```diff
@@ -114,7 +120,8 @@ jar {
   }
   duplicatesStrategy = 'exclude'
   destinationDirectory = file('dist/')
-  archiveBaseName = ant.properties['version-name']
+  archiveBaseName = "KoLmafia"
+  classifier = "${versioning.info.dirty ? 'Modified' : ''}"
```

Comment:
classifier is deprecated in favor of archiveClassifier.

How will the resulting file look? KoLmafia-r20999-Modified.jar?

(preference nit: make this lowercase?)

Reply:
I left it `KoLmafia` since that matches the current filename case from the ant build (locally modified for ant is KoLmafia-r###M.jar)

Reply:
Oh, I meant specifically to make "Modified" lowercase. But I'm fine with it either way.